cmd/acra-server/acra-server.go

```diff
@@ -147,7 +147,8 @@ func realMain() error {
 	scriptOnPoison := flag.String("poison_run_script_file", "", "On detecting poison record: log about poison record detection, execute script, return decrypted data")
 
 	withZone := flag.Bool("zonemode_enable", false, "Turn on zone mode")
-	enableHTTPAPI := flag.Bool("http_api_enable", false, "Enable HTTP API")
+	enableHTTPAPI := flag.Bool("http_api_enable", false, "Enable HTTP API. Use together with --http_api_use_tls whenever possible.")
+	httpAPIUseTLS := flag.Bool("http_api_use_tls", false, "Enable HTTPS support for the API. Use together with the --http_api_enable. TLS configuration is the same as in the Acra Proxy.")
```

Comment:
what about to renam it from `http_api_use_tls` to `http_api_tls_transport_enable`? we have a lot of flags with `_enable` ending to show that it turns something on. Plus, we have `vault_tls_transport_enable` flag. So, what about to rename to be consistent with other flag style?